packages/svelte/src/internal/client/dom/task.js

```diff
@@ -1,33 +1,63 @@
 import { run_all } from '../../shared/utils.js';
 
-let is_task_queued = false;
+// Fallback for when requestIdleCallback is not available
+const request_idle_callback =
+	typeof requestIdleCallback === 'undefined'
+		? (/** @type {() => void} */ cb) => setTimeout(cb, 1)
+		: requestIdleCallback;
```

Comment:
We can be sure it's always available in the browser, right? I feel like this patch should move into our test suite somewhere so it doesn't affect bundle size.

Reply:
Safari doesn't yet have support for it.

Reply:
Oh I didn't know that, nevermind then